pyaedt/hfss.py

```diff
@@ -3686,6 +3703,106 @@ def assign_impedance_to_sheet(self, sheet_name, sourcename=None, resistance=50,
             return self._create_boundary(sourcename, props, "Impedance")
         return False
 
+    @pyaedt_function_handler()
+    def assign_anisotropic_impedance_to_sheet(
+        self,
+        sheet_name,
+        sourcename=None,
+        ZxxResi=0,
+        ZxxReac=0,
+        ZxyResi=0,
+        ZxyReac=0,
+        ZyxResi=0,
+        ZyxReac=0,
+        ZyyResi=0,
+        ZyyReac=0,
+        is_infground=False,
+        coord_sys="Global",
+    ):
+        """Create an anisotropic impedance taking one sheet.
+
+        Parameters
+        ----------
+        sheet_name : str
+            Name of the sheet to apply the boundary to.
+        sourcename : str, optional
+            Name of the anisotropic impedance. The default is ``None``.
+        ZxxResi : optional
+            Resistance value of xx component in ohms. The default is ``0``. If ``None``,
+            this parameter is disabled.
+        ZxxReac : optional
+            Reactance value of xx component in ohms. The default is ``0``. If ``None``,
+            this parameter is disabled.
+        ZxyResi : optional
+            Resistance value of xy component in ohms. The default is ``0``. If ``None``,
+            this parameter is disabled.
+        ZxyReac : optional
+            Reactance value of xy component in ohms. The default is ``0``. If ``None``,
+            this parameter is disabled.
+        ZyxResi : optional
+            Resistance value of yx component in ohms. The default is ``0``. If ``None``,
+            this parameter is disabled.
+        ZyxReac : optional
+            Reactance value of yx component in ohms. The default is ``0``. If ``None``,
+            this parameter is disabled.
+        ZyyResi : optional
+            Resistance value of yy component in ohms. The default is ``0``. If ``None``,
+            this parameter is disabled.
+        ZyyReac : optional
+            Reactance value of yy component in ohms. The default is ``0``. If ``None``,
+            this parameter is disabled.
+        is_infground : bool, optional
+            Whether the impedance is an infinite ground. The default is ``False``.
+        coord_sys : optional
+            The name of the coordinate system for the xy plane. The default is ``Global``.
```

Comment:
```suggestion
            Name of the coordinate system for the xy plane. The default is ``"Global"``.
```
We omit the article (The, An, A) at the beginning of the brief description. Also, a string default is enclosed in both backticks and quotation marks.)